changes/next/sieve-unicode-casemap

```diff
@@ -0,0 +1,14 @@
+
+Description:
+
+Adds support for comparator-i;unicode-casemap (RFC 9394) to Sieve
+
```

Comment:
As far as I see, RFC 9394 is not about Sieve and not about Unicode.

Reply:
Agreed, that looks like it should be 5051 ?

Reply:
Fixed